homeassistant/components/notify/html5.py

```diff
@@ -60,6 +63,7 @@ def get_service(hass, config):
 
     hass.wsgi.register_view(
         HTML5PushRegistrationView(hass, registrations, json_path))
+    hass.wsgi.register_view(HTML5PushCallbackView(hass, json_path))
```

Comment:
We keep `registrations` in sync with the file at `json_path`, so we should not have to load it again ?